packages/nodes-base/nodes/Airtop/actions/extraction/getPaginated.operation.ts

```diff
@@ -67,18 +70,21 @@ export const description: INodeProperties[] = [
 				name: 'paginationMode',
 				type: 'options',
 				default: 'auto',
-				hint: "The mode to use for pagination. If set to 'Auto', Airtop AI first looks for pagination links and then tries infinite scrolling. If set to 'Paginated', it uses pagination links. If set to 'Infinite Scroll', it scrolls the page to load more content.",
+				description: 'The pagination approach to use',
 				options: [
 					{
 						name: 'Auto',
+						description: 'Look for pagination links first, then tries infinite scrolling',
```

Comment:
For consistency since others are in present tense:
```suggestion
						description: 'Look for pagination links first, then try infinite scrolling',
```